pkg/scheduler/framework/plugins/noderesources/balanced_allocation.go

```diff
@@ -88,7 +88,7 @@ func balancedResourceScorer(requested, allocable resourceToValueMap, includeVolu
 		return 0
 	}
 
-	if includeVolumes && utilfeature.DefaultFeatureGate.Enabled(features.BalanceAttachedNodeVolumes) && allocatableVolumes > 0 {
+	if includeVolumes && allocatableVolumes > 0 {
```

Comment:
do we not need to test the flag anymore?

Reply:
we don't, the calling function only sets `includeVolumes` if the flag is enabled. Forgot to carry that comment over https://github.com/kubernetes/kubernetes/pull/98583/files#diff-6fe34bf325186cb8568837e8c946367248be19d153f9e4ce1b05b1d3e66ecc10R109